examples/with-apollo/components/PostList.js

```diff
@@ -4,20 +4,17 @@ import PostUpvoter from './PostUpvoter'
 
 const POSTS_PER_PAGE = 10
 
-// The data prop, which is provided by the HOC below contains
-// a `loading` key while the query is in flight
 function PostList (props) {
-  const { data: { allPosts, loading, _allPostsMeta }, loadMorePosts } = props
-  if (loading) {
+  if (props.data.loading) {
```

Comment:
I meant the opposite: function( { data: { allPosts, loading, _allPostsMeta }, loadMorePosts } ) { ...

But anyway nothing critical